tests/IntegrationTests/FlureeTransactionTests.cs

```diff
@@ -18,12 +18,12 @@ namespace IntegrationTests
     [Trait("Category", "Transact")]
     public class FlureeTransactionTests
     {
-        private readonly IFlurlClient _flurlClient;
         private readonly IFlureeCommandService _flureeCommandService;
-        public FlureeTransactionTests(IFlurlClientFactory factory, IConfiguration configuration, IFlureeCommandService flureeCommandService)
+        public FlureeTransactionTests()
         {
-            _flurlClient = factory.Get(configuration["fluree"]);
-            _flureeCommandService = flureeCommandService;
+            _flureeCommandService = new FlureeCommandService(
+                new PerBaseUrlFlurlClientFactory(),
+                "http://localhost:8090");
```

Comment:
Why do this and not do the factory config I had in orginaly? The factory can still make a perbaseurl request with the ip given to it from the config